### PR TITLE
css-box-model-Laura Grace Manimtim

### PR DESCRIPTION
**Pull request by lauragracem**
Screen Shot 2022-05-11 at 11 23 53 AM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-left: 12px;
+  margin-right: 16px;
```
Comment (TimelikeClosure) [inline css-box-model/styles.css:5]: image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by lauragracem** [comment]
Screen Shot 2022-05-11 at 11 34 00 AM

File: css-box-model/styles.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin: 16px 12px;
```
Comment (TimelikeClosure) [inline css-box-model/styles.css:4]: The instructions don't say anything about top or bottom margin:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by lauragracem** [comment]
Screen Shot 2022-05-11 at 11 42 24 AM

File: css-box-model/styles.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin: 0 16px 0 12px;
```
Comment (TimelikeClosure) [inline css-box-model/styles.css:4]: Please don't manually set the top or bottom margin. I suggest avoiding the `margin` shorthand altogether.

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-box-model/styles.css (hunk 3 above)
Comment (lauragracem) [reply css-box-model/styles.css:4]: Okay let me try again

**Comment by lauragracem** [comment]
Oh wait, don't count that last one. I didn't resolve anything yet 🙈

**Comment by lauragracem** [comment]
Screen Shot 2022-05-11 at 11 53 00 AM

**Review by TimelikeClosure — APPROVED** [review]
